Makefile

```diff
@@ -79,7 +79,7 @@ vendor:
 	go mod verify
 
 .PHONY: generate
-generate: build-jsonnet manifests/0000_50_cluster-monitoring-operator_02-role.yaml docs
+generate: build-jsonnet manifests/0000_50_cluster-monitoring-operator_02-role.yaml docs check-assets
```

Comment:
I don't think it is meaningful to have this rule as part of `generate` as we don't generate anything. It could be added to the `format` one as you added it in the formatting section, but I don't think if that would be a good fit either. Should we maybe add a `verify` rule for this kind of verification?

Reply:
I agree this isn't the best place and we should consider shuffling make targets a bit. However I put it into `generate` only because this is the only target which was related to checked assets **and** it is running in CI.

Reply:
SGTM

Reply:
opened https://issues.redhat.com/browse/MON-1656